recipes/renode-cli/recipe.yaml

```diff
@@ -0,0 +1,83 @@
+context:
+  name: renode-cli
+  version: "1.15.3"
+  python_min: "3.6"
```

Comment:
This variable is not used anywhere, and furthermore the oldest support version of Python is 3.9 - 3.6 has been EOL for quite a while now